osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -53,7 +54,9 @@ def create_job(self, extraction_order):
         bounding_geometry = extraction_order.excerpt.bounding_geometry.subclass_instance
 
         request_data = OrderedDict({
-            "callback_url": "http://example.com",
+            "callback_url": "http://example.com{0}".format(
```

Comment:
No, we don't (at least _not really_ :tm: ). In the case of running within docker in production, we _could_ take it from the `VIRTUAL_HOST` docker environment variable, but that is very specific how we run this image(s).

Two options: 
- defining the a site in Django's site framework, and take it from there. This would also be trivial to extend the entrypoint script for the docker-container, and setting that `VIRTUAL_HOST` (or any other environment entry that suits us) to the corresponding [site](https://docs.djangoproject.com/en/1.8/ref/contrib/sites/) and [SITES_ID](https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-SITE_ID) to 1.
- somehow reaching/getting to the request and then using django-rest-frameworks [reverse](http://www.django-rest-framework.org/api-guide/reverse/#reverse), which returns the fully qualified url.